web-app/js/portal/ui/openlayers/layer/NcWMS.js

```diff
@@ -102,9 +102,7 @@ OpenLayers.Layer.NcWMS = OpenLayers.Class(OpenLayers.Layer.WMS, {
         });
     },
 
-    _timeSeriesDatesLoaded: function(datesWithData) {
```

Comment:
I'm with you. At the time of doing that work it was a bit of a stretch even getting done what was required - at that time frame.

Portal code needs refactoring all over the place - that's yet another one. Not sure when we can actually get to it though :/